src/coreclr/src/jit/target.h

```diff
@@ -6,10 +6,11 @@
 #ifndef TARGET_H_
 #define TARGET_H_
 
-#if defined(TARGET_UNIX)
-#define FEATURE_VARARG 0
-#else
+ // Native Varargs are not supported on Unix (all architectures) and Windows ARM
+#if (TARGET_WINDOWS && !TARGET_ARM)
```

Comment:
Nit: Everything else in this file uses `defined(TARGET_ARM)`, etc.

Reply:
Ah, yes. Sorry, you mentioned it already but didn't realized `defined` part. Will fix it.